### PR TITLE
Adding references to PredictionEnginePool in Tutorials

### PR DESCRIPTION
**Pull request by luisquintanilla**
Adding note to tutorials advising users to use `PredictionEnginePool` in multi-threaded production environments.

| Tutorial | Preview Link | Complete |
|---|---|---|
| Sentiment Analysis | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis?branch=pr-en-us-14831 | Y |
| GitHub Issue Classification | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification?branch=pr-en-us-14831 | Y |
| Predict Prices | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/predict-prices?branch=pr-en-us-14831 | Y |
| Iris Clustering |  https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/iris-clustering?branch=pr-en-us-14831| Y |
| Movie Recommendation | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommendation?branch=pr-en-us-14831 | Y |
| Image Classification | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification?branch=pr-en-us-14831 | Y |
| Text Classification |https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/text-classification-tf?branch=pr-en-us-14831  | Y |

[AB#1594322](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1594322)

### CODE REVIEW

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 1]
```diff
@@ -280,13 +280,16 @@ Use the following code to display the metrics:
...
 
-    The [PredictionEngine](xref:Microsoft.ML.PredictionEngine%602) is a convenience API, which allows you to pass in and then perform a prediction on a single instance of data.
+    The [PredictionEngine](xref:Microsoft.ML.PredictionEngine%602) is a convenience API, which allows you to pass in and then perform a prediction on a single instance of data. [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602) is not thread-safe. It's acceptable to use in single-threaded or prototype environments. For improved performance and thread safety in production environments, use the `PredictionEnginePool` service, which creates an [`ObjectPool`](xref:Microsoft.Extensions.ObjectPool.ObjectPool%601) of [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602) objects for use throughout your application. See this guide on how to [use `PredictionEnginePool` in an ASP.NET Core Web API](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/serve-model-web-api-ml-net#register-predictionenginepool-for-use-in-the-application)
```
Comment (natke) [inline docs/machine-learning/tutorials/sentiment-analysis.md]:
```suggestion
    The [PredictionEngine](xref:Microsoft.ML.PredictionEngine%602) is a convenience API, which allows you to perform a prediction on a single instance of data. [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602) is not thread-safe. It's acceptable to use in single-threaded or prototype environments. For improved performance and thread safety in production environments, use the `PredictionEnginePool` service, which creates an [`ObjectPool`](xref:Microsoft.Extensions.ObjectPool.ObjectPool%601) of [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602) objects for use throughout your application. See this guide on how to [use `PredictionEnginePool` in an ASP.NET Core Web API](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/serve-model-web-api-ml-net#register-predictionenginepool-for-use-in-the-application)
```

**Review by natke — COMMENTED** [review]
Looks great!  One minor suggestion

**Review by natke — COMMENTED** [review]
Looks good. Tiny comment: one of the instances splits the content into two paras. Is two better than one? If yes, we should change. If not, we should keep it consistent.

**Comment by luisquintanilla** [comment]
I split it into two for clarification. One saying to use the `PredictionEngine` and `Predict` method. And then the snippet giving more context on `PredictionEngine`. I can merge the two though.

**Review by natke — APPROVED** [review]

**Pull request merged**